docs/docs_skeleton/docs/modules/agents/how_to/custom_llm_agent.mdx

```diff
@@ -1,13 +1,13 @@
-# Custom LLM Agent
+# Custom LLM agent
 
 This notebook goes through how to create your own custom LLM agent.
 
 An LLM agent consists of three parts:
 
-- PromptTemplate: This is the prompt template that can be used to instruct the language model on what to do
+- `PromptTemplate`: This is the prompt template that can be used to instruct the language model on what to do
 - LLM: This is the language model that powers the agent
```

Comment:
Maybe `LLM` to consistency?

Reply:
I don't see how this would help since it's not a class/variable name or anything. LLM without backticks is consistently used throughout the docs so I think this would be confusing and inconsistent.

Reply:
That's OK. I'd prefer **formatting** context where each paragraph is started with `term`. If you prefer the **class** context there all class names are formatted as `ClassName`, that's not big deal.